perceval/backends/core/groupsio.py

```diff
@@ -265,15 +270,15 @@ def _write_archive(r, filepath):
         with open(filepath, 'wb') as fd:
             fd.write(r.raw.read())
 
-    def __find_group_id(self):
-        """Find the id of a group given its name by iterating on the list of subscriptions"""
-
+    def __find_group_info(self):
+        """Find the id and download archive permission of a group given
```

Comment:
Please follow the format:
```
"""Find the id ....
....
"""
group_subscriptions = self.subscriptions()
```

 Example at https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/github.py#L273

Reply:
Good Catch!